apps/icelandic-names-registry/backend/src/app/modules/icelandic-name/icelandic-name.controller.ts

```diff
@@ -53,15 +54,15 @@ export class IcelandicNameController {
     return await this.icelandicNameService.getAll()
   }
 
-  @Get(':id')
+  @Get('id/:id')
```

Comment:
Not sure this falls inline with the API design guide (notably the naming conventions).
This doesn't specify what resource you're fetching and isn't really a RESTful design.
I personally haven't used this API before, so I'm assuming this is fetching Icelandic names, thus the following would make more sense for URI structure:
```
api/icelandic-names-registry/names/:id
api/icelandic-names-registry/names?search=xxx
```

Also feels like initial-letter can be combined with search ... it's just a matter of skipping `%` in the database query.